_posts/en/newsletters/2021-11-10-newsletter.md

```diff
@@ -0,0 +1,225 @@
+  minimum feerate is 3sat/vB, an individual transaction with a 1sat/vB feerate may
+  be accepted. ATMP is called with `bypass_limits` during a [reorg][atmp bypass
+  limits]; these transactions may have low individual feerates but high descendant
+  feerates. The size of the transactions is limited to
+  `MAX_DISCONNECTED_TX_POOL_SIZE`, or 20MB."
+  a3link="https://bitcoincore.reviews/23173#l-132"
+%}
+
+## Preparing for taproot #21: thank you!
+
+*The final entry in a weekly [series][series preparing for taproot]
+about how developers and service providers can prepare for the upcoming
+activation of taproot at block height {{site.trb}}.*
+
+{% include specials/taproot/en/21-thanks.md %}
+
+## Releases and release candidates
+
+*New releases and release candidates for popular Bitcoin infrastructure
+projects.  Please consider upgrading to new releases or helping to test
+release candidates.*
+
+- [BTCPay Server 1.3.3][] is a release <!-- along with the previous
+  release one day earlier --> containing a critical security patch for
+  instances on a shared server which also share their LN nodes.  Also
+  included are minor features and other bug fixes.
+
+- [Rust-Lightning 0.0.103][] is a release which adds an
+  `InvoicePayer` API for retrying payments when some paths fail.
+
+- [C-Lightning 0.10.2rc2][] is a release candidate that [includes][decker
+  tweet] a fix for the [uneconomical outputs security issue][news170
+  unec bug], a smaller database size, and an improvement in the
+  effectiveness of the `pay` command (see the *notable changes* section
+  below).
+
+- [LND 0.14.0-beta.rc2][] is a release candidate that includes
+  additional [eclipse attack][topic eclipse attacks] protection (see
+  [Newsletter #164][news164 ping]), remote database support ([Newsletter
+  #157][news157 db]), faster pathfinding ([Newsletter #170][news170
+  path]), improvements for users of Lightning Pool ([Newsletter
+  #172][news172 pool]), and reusable [AMP][topic amp] invoices
+  ([Newsletter #173][news173 amp]) in addition to many other features
+  and bug fixes.
+
+## Notable code and documentation changes
+
+*Notable changes this week in [Bitcoin Core][bitcoin core repo],
+[C-Lightning][c-lightning repo], [Eclair][eclair repo], [LND][lnd repo],
+[Rust-Lightning][rust-lightning repo], [libsecp256k1][libsecp256k1
+repo], [Hardware Wallet Interface (HWI)][hwi repo],
+[Rust Bitcoin][rust bitcoin repo], [BTCPay Server][btcpay server repo],
+[BDK][bdk repo], [Bitcoin Improvement Proposals (BIPs)][bips repo], and
+[Lightning BOLTs][bolts repo].*
+
+- [Rust-Lightning #1078][] adds `channel_type` negotiation as defined in [BOLTs
+  #880][] and covered in [Newsletter #165][news165 bolts-880]. This
+  implementation does not currently send the proposed feature bit from [BOLTs
+  #906][], which are likely to be required for turbo channels, but if not, will
+  be needed for anchor channels.
```

Comment:
BOLTs 906 isn't required for turbo channels/anchor outputs. When Matt says "It is likely to be required for turbo channels [...]", he's referring to support for BOLTs 880.

```suggestion
  #906][]. [BOLTs #880][] is required for [anchor channels][topic anchor outputs] and may also be needed to support [turbo channels](https://bitcoinops.org/en/newsletters/2021/07/07/#zero-conf-channel-opens).
```